recipes/jupyter_enterprise_gateway/meta.yaml

```diff
@@ -0,0 +1,66 @@
+
+{% set name = "jupyter_enterprise_gateway" %}
+{% set version = "0.7.0" %}
+{% set md5 = "31763073af6e6e405e0b0cefea49fe7e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  noarch: python
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - jupyter-enterprisegateway = enterprise_gateway:launch_instance
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - jupyter_core >=4.0
+    - jupyter_client >=4.2.0
+    - notebook >=5.1.0,<6.0
+    - jupyter_kernel_gateway >=1.1.2
+    - traitlets >=4.2.0
+    - tornado >=4.2.0
+    - requests >=2.7,<3.0
+    - paramiko >=2.1.2
+    - pexpect
+
+test:
+  imports:
+    - enterprise_gateway
+    - enterprise_gateway.services
+    - enterprise_gateway.services.kernels
+    - enterprise_gateway.services.kernelspecs
+    - enterprise_gateway.services.sessions
+  commands:
+    - jupyter-enterprisegateway --help
+    - jupyter enterprisegateway --help
+
+about:
+  home: https://github.com/jupyter-incubator/jupyter_enterprise_gateway
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE.md
+  summary: 'A web server for spawning and communicating with remote Jupyter kernels'
+
+  description: |
+    Jupyter Enterprise Gateway is a lightweight, multi-tenant, scalable and secure gateway that enables
+    Jupyter Notebooks to share resources across an Apache Spark cluster
+  doc_url: http://jupyter-enterprise-gateway.readthedocs.io/en/latest/
+  dev_url: https://github.com/jupyter-incubator/jupyter_enterprise_gateway
+
+extra:
+  recipe-maintainers:
+    - liukun1016
+    - lresende
```

Comment:
@lresende, are you happy to be a maintainer for the conda-forge package of jupyter_enterprise_gateway?